### PR TITLE
[stable/yugabyte][stable/yugaware] Prom regex update

### PR DESCRIPTION
**Pull request by Arnav15**
#### What this PR does / why we need it:
Yugabyte added new percentile metrics which needs the regex for pulling handler_latency metrics to work without the sum/count suffix.

### CODE REVIEW

File: stable/yugabyte/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: yugabyte
-version: 2.0.9
-appVersion: 2.0.9.0-b13
+version: 2.0.11
+appVersion: 2.0.11.0-b23
```
Comment (bmatican) [inline stable/yugabyte/Chart.yaml]: does this version have the percentile changes in YB?

File: stable/yugaware/templates/service.yaml [hunk 2]
```diff
@@ -111,10 +109,12 @@ spec:
               subPath: postgres_data
         - name: prometheus
           image: prom/prometheus:v2.2.1
+          securityContext:
```
Comment (bmatican) [inline stable/yugaware/templates/service.yaml]: what's this about?

File: stable/yugaware/templates/service.yaml [hunk 3]
```diff
@@ -111,10 +109,12 @@ spec:
...
+            runAsUser: 0
           volumeMounts:
           - name: prometheus-config
             mountPath: /etc/prometheus/
-          - name: prometheus-storage
+          - name: yugaware-storage
```
Comment (bmatican) [inline stable/yugaware/templates/service.yaml]: @ramkumarvs should prolly eyeball this

Comment (ramkumarvs) [reply stable/yugaware/templates/service.yaml]: LG. we chatted about this.

File: stable/yugaware/templates/service.yaml (hunk 2 above)
Comment (ramkumarvs) [reply stable/yugaware/templates/service.yaml]: @Arnav15 are we missing some more info here?

**Review by ramkumarvs — APPROVED** [review]
LG

**Pull request merged**